libs/scala.json

```diff
@@ -1,12 +1,15 @@
 [
   {"id":"scala.akka","imports":["akka.actor.","akka.event.","akka.camel.","akka.cluster."],"name":"Akka","repo":"akka/akka/tree/master/akka-actor","tags":["actor-model"],"tech":["app-arch"],"status":"model-ready"},
   {"id":"scala.akka-streams","imports":["akka.stream."],"name":"Akka Streams","repo":"akka/akka/tree/master/akka-stream","tags":["stream-processing"],"tech":["app-arch"],"status":"model-ready"},
+  {"id":"scala.sangria","imports":["sangria."],"name":"Sangria","repo":"sangria-graphql/sangria","tags":["api-framework"],"tech":["app-arch"],"status":"awaiting-model"},
   {"id":"scala.circe","imports":["io.circe."],"name":"Circe","repo":"circe/circe","tags":["json","json-transformation"],"tech":["app-data"],"status":"model-ready"},
   {"id":"scala.json4s","imports":["org.json4s."],"name":"json4s","repo":"json4s/json4s","tags":["json-parser"],"tech":["app-data"],"status":"model-ready"},
   {"id":"scala.cats","imports":["cats."],"name":"Cats","repo":"typelevel/cats","tags":["monads"],"tech":["functional"],"status":"model-ready"},
   {"id":"scala.akka-http","imports":["akka.http."],"name":"Akka HTTP","repo":"akka/akka-http","tags":["framework"],"tech":["scala-web"],"status":"model-ready"},
   {"id":"scala.play","imports":["play."],"name":"Play Framework","repo":"playframework/playframework","tags":["framework"],"tech":["scala-web"],"status":"model-ready"},
   {"id":"scala.scalatra","imports":["org.scalatra."],"name":"Scalatra","repo":"scalatra/scalatra","tags":["framework"],"tech":["scala-web"],"status":"model-ready"},
+  {"id":"scala.silhouette","imports":["com.mohiva.play.silhouette.","com.mohiva.silhouette."],"name":"Silhouette","repo":"mohiva/play-silhouette","tags":["framework"],"tech":["scala-web"],"status":"awaiting-model"},
```

Comment:
This looks like an authentication library, so let's make tags = ["auth"]

Reply:
Will do it